### PR TITLE
README: add a note how people can complain

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
README: add a note how people can complain

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

**Comment by Andersson007** [comment]
@Jorge-Rodriguez could you please check the wording?

**Comment by Andersson007** [comment]
As it's described in https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations , I'll close it, thanks

**Comment by Andersson007** [comment]
@Jorge-Rodriguez changed, i think this could be difficult to reach at the bottom of the CoC, so I put it in README explicitly what do you think?

**Review by Jorge-Rodriguez — APPROVED** [review]
Great idea! 👍

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 We follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within this project.
 
+If you encounter abusive behavior violating [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html), please refer to the [policy violations](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations) Code of Conduct section how to complain.
```
Comment (Jorge-Rodriguez) [inline README.md]:
```suggestion
If you encounter abusive behavior violating the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html), please refer to the [policy violations](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations) section of the Code of Conduct for information on how to raise a complaint.
```

**Review by Jorge-Rodriguez — COMMENTED** [review]
Updated wording.

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md]: that's better, thanks! committed

**Comment by Andersson007** [comment]
@Jorge-Rodriguez thanks for reviewing!

**Pull request merged**